src/main/java/com/cedarsoftware/util/io/factory/EnumClassFactory.java

```diff
@@ -7,50 +7,68 @@
 
 import java.util.Optional;
 
+/**
+ * Factory class to create Throwable instances.  Needed for JDK17+ as the only way to set the
```

Comment:
I think the comment's first sentence is a copy-paste error - should it read "Factory class to create Enum instances."  ?